pkg/webui/oauth/api/index.js

```diff
@@ -27,7 +29,7 @@ const instance = axios.create({
 export default {
   users: {
     async register(userData) {
-      return axios.post(`/api/v3/users`, userData)
+      return axios.post(`${isBaseUrl}/users`, userData)
```

Comment:
what about `resetPassword` and `updatePassword` ?

Reply:
Ha. Missed them after the rebase.